src/LinearOperators.jl

```diff
@@ -9,7 +9,7 @@ export AbstractLinearOperator, LinearOperator,
        opInverse, opCholesky, opLDL, opHouseholder, opHermitian,
        check_ctranspose, check_hermitian, check_positive_definite,
        shape, hermitian, ishermitian, symmetric, issymmetric,
-       opRestriction, opExtension
+       opRestriction, opExtension, opBypass
```

Comment:
@abelsiqueira, `opBypass` is an `opEye` now, you don't need to export a `opBypass` function?

Reply:
Thanks